English/Grades/Middle.md

```diff
@@ -1,43 +1,47 @@
-- Стадия миддла самая энергозатратная с точки зрения обучения. От вас требуется не только прокачивать технические скиллы, но также навыки коммуникации и погружение в проблемы бизнеса. Это значит, что вам требуется одновременно развиваться в нескольких направлениях одновременно. Уделяйте внимание в равной степени как "хард", так и "софт" скиллам.
-- Должное внимание "софт" скиллам также повышает вероятность того, что вы быстрее станете востребованным профессионалом на рынке. Вы можете попытаться стать узконаправленным техническим специалистом и игнорировать коммуникативные навыки, но, во-первых, компаниям нечасто нужны подобные кадры в больших количествах, во-вторых, вам придется конкурировать с лучшими из лучших. Если вы действительно готовы состязаться с лучшими специалистами на рынке, то смело идите вперед, но все же рекомендуем подумать о диверсификации своих навыков. 
+- Can personally make decisions based on technical knowledge/background of a project
+- Understands when a solution is "good enough" to prevent overengineering
+- Has a team player mindset
+- Can formulate and share an opinion with other teammates
+- Has empiric background with different methodologies: Kanban, Agile/Scrum, Waterfall, etc.
+- Helps other teammates
 
 
-### Про опыт
-- Основная ловушка для многих мидлов: фанбойство по технологиям, фреймворкам, внедрением паттернов или подходам к разработке. Постарайтесь прагматично подходить к выполнению задач на проекте. Не нужно пытаться затянуть все последние новинки, только чтобы поиграться с ними или ради строчки в резюме. На этом этапе очень велик соблазн проявить свое мастерство через обилие технологий или оверинжиниринг.
-- Если вы действительно считаете, что проекту нужна новая библиотека или фреймворк - обсудите это с синьором/тимлидом. Предложите им попробовать создать прототип, где сможете проверить их в действии прежде чем втягивать в проект. Пожалуйста, никогда не втягивайте их в обход всей команды или втихаря! Для вас - это развлечение, но для проекта/тимлида это станет головной болью в будущем, т.к. это повысит стоимость поддержки проекта и принесёт неожиданные проблемы. 
+## Tips and recommendations
+
+### Studying
+
+- It's time to improve soft skills if you want to become a Senior one. The technical expertise goes a bit behind and an ability to build dialogs and find compromises with others go first. A good developer is not one who writes a lot of code, but understands how to solve a problem efficiently with minimal "bleeding". Good - without any new code, ideally - if even tens/hundred lines of code are removed.
```

Comment:
Good - without any new code, ideally - if even tens/hundred lines of code are removed. -> It is better if you can solve a problem without any new code. It is ideal if you can even remove tens/hundreds lines of code